types/provider.go

```diff
@@ -14,14 +14,22 @@ type Provider struct {
 
 // ProviderType lists available cloud providers.
 type ProviderType string
+type ProfileType string
+type SeedType string
 
 const (
-	// GCP stands for the Google Cloud Platform.
-	GCP ProviderType = "gcp"
-	// Azure stands for the Microsoft Azure Cloud Computing Platform.
-	Azure ProviderType = "azure"
-	// AWS stands for Amazon Web Services.
-	AWS ProviderType = "aws"
+	// GCPProvider stands for the Google Cloud Platform provider.
+	GCPProvider ProviderType = "gcp"
```

Comment:
Is there a reason to change all these constants? We already defined the API like this for a reason, unless there is a need to change this, please revert it.

Reply:
done..reverted